src/pool/srv_target.c

```diff
@@ -237,21 +244,28 @@ pool_alloc_ref(void *key, unsigned int ksize, void *varg,
 		goto err_iv_lock;
 	}
 
-	if (arg->pca_need_group) {
-		char id[DAOS_UUID_STR_SIZE];
+	uuid_unparse_lower(key, group_id);
+	rc = crt_group_secondary_create(group_id, NULL /* primary_grp */,
+					NULL /* ranks */, &pool->sp_group);
```

Comment:
so we still export the cart group to daos?

Reply:
Yes. DAOS needs the CaRT group pointer to send CoRPCs and create IV NSs. But going forward, DAOS servers shall only get this pool group pointer through ds_pool, not through crt_group_lookup (which does not take a group reference).